cgrun059: Uncaught exception ghc-internal:GHC.Internal.Exception.ErrorCall:

Error: File not found

HasCallStack backtrace:
  collectBacktraces, called at libraries/ghc-internal/src/GHC/Internal/Exception.hs:91:13 in ghc-internal:GHC.Internal.Exception
  toExceptionWithBacktrace, called at libraries/ghc-internal/src/GHC/Internal/Exception.hs:118:5 in ghc-internal:GHC.Internal.Exception
  error, called at cgrun059.hs:12:28 in main:Main

